waku/v2/protocol/filter/filter_health_check.go

```diff
@@ -17,17 +19,23 @@ func (wf *WakuFilterLightNode) PingPeers() {
 }
 
 func (wf *WakuFilterLightNode) PingPeer(peer peer.ID) {
-	ctxWithTimeout, cancel := context.WithTimeout(wf.CommonService.Context(), wf.peerPingInterval)
+	ctxWithTimeout, cancel := context.WithTimeout(wf.CommonService.Context(), PingTimeout)
```

Comment:
Should we get rid of `wf.peerPingInterval` ?

Reply:
that is still required as it is used in https://github.com/waku-org/go-waku/blob/8cfe01d7d4776c41053aa2c8ca435b58469730e0/waku/v2/protocol/filter/filter_health_check.go#L45